var/spack/repos/builtin/packages/fenics-basix/package.py

```diff
@@ -17,6 +17,6 @@ class FenicsBasix(CMakePackage):
     version("main", branch="main")
     version("0.1.0", sha256="2ab41fe6ad4f6c42f01b17a6e7c39debb4e0ae61c334d1caebee78b741bca4e7")
 
-    depends_on("cmake@3.16:", type="build")
-    depends_on("xtensor@0.23.4:", type="build")
+    depends_on("cmake@3.18:")
+    depends_on("xtensor@0.23.10:")
```

Comment:
The `type=build` should stay (unless this application links to some CMake internal library)

Reply:
`xtensor` appears in the public interface, so is needed to link against Basix.

See below on why using `type=build` often leads to problems with CMake due to CMake version mis-matches.